test/integration/api/oidc.js

```diff
@@ -0,0 +1,38 @@
+const { testService } = require('../setup');
+
+describe('api: /oidc/...', () => {
+  if (process.env.TEST_AUTH === 'oidc') {
+    describe('GET /oidc/login', () => {
+      it('should redirect to IdP if no parameters are provided', testService(service =>
+        service.get('/v1/oidc/login')
+          .expect(307)
+          .then(({ text, headers }) => {
+            const expectedUrlPrefix = 'http://localhost:9898/auth?client_id=odk-central-backend-dev&scope=openid%20email&response_type=code&redirect_uri=http%3A%2F%2Flocalhost%3A8989%2Fv1%2Foidc%2Fcallback&resource=http%3A%2F%2Flocalhost%3A8989%2Fv1&code_challenge=';
```

Comment:
Once #1392 is merged, it will be seen that this URL has changed in subtle ways.  Confirm in the `openid-client` repo why this is and if it's significant.